jekyll/_cci2/language-clojure.md

```diff
@@ -40,7 +40,7 @@ jobs: # basic units of work in a run
   build: # runs not using Workflows must have a `build` job as entry point
     working_directory: ~/cci-demo-clojure # directory where steps will run
     docker: # run the steps with Docker
-      - image: circleci/clojure:lein-2.9.1 # ...with this image as the primary container; this is where all `steps` will run
+      - image: cimg/clojure:lein-2.9.1 # ...with this image as the primary container; this is where all `steps` will run
```

Comment:
The tag `lein-2.9.1` doesn't exist. We can use `1.10`.

Reply:
Fixed